src/apply.jl

```diff
@@ -153,5 +171,5 @@ _postformat(::Cardinality, result, A, append_dim) = result
 function _postformat(::ManyToOne, result, A, append_dim)
     new_size = collect(size(A))
     setindex!(new_size, 1, dim(A, append_dim))
-    return reshape(result, new_size...)
+    return copy(reshape(result, new_size...))  # return a copy to remove the reshape type
```

Comment:
Was the reshape type a problem before? Just wondering why we weren't already returning a copy.

Reply:
Yeah it was a problem because calling `parent` on it later on removed the `ReshapeArray` type and returned the underlying `AxisArray` / `KeyedArray` (which is what we actually wanted to call `parent` on). So calling `copy` just performs that step in advance.